### PR TITLE
Feat: Mobile Improvements

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
Fix some mobile view bugs and focus animation:
- Removes hover on touch devices
- Fixes ProductCard's border-radius
- Updates ProductCard's focus state animation

## How it works? 
|Improvements|Before|After|
|-|-|-|
|Image Borders|Image from iOS|IMG_F3E03AB3D67C-1|
|Breadcrumbs|Screen Shot 2022-01-11 at 11 35 46|image|

## How to test it?
Open the [Main Styled branch preview](https://deploy-preview-131--basestore.netlify.app) on your mobile phone and compare it with [this preview link here](https://preview-214--base.preview.vtex.app).

### CODE REVIEW

File: src/components/product/ProductCard/product-card.scss [hunk 1]
```diff
@@ -5,22 +5,25 @@
...
+    &:hover {
+      border: 1px solid var(--color-border-display);
+      box-shadow: var(--box-shadow);
+    }
   }
 
   &:focus,
   &:focus-within {
-    outline: 2px solid var(--bg-focus-ring);
+    box-shadow: 0 0 0 1px var(--color-neutral-0), 0 0 0 var(--border-width-3) var(--bg-focus-ring);
```
Comment (hellofanny) [inline src/components/product/ProductCard/product-card.scss:26]: We are also using this on `sku-selector.scss`. Do you think we will need to apply this also to other components? If so, we could extract it to a mixin in the future.

Comment (hellofanny) [inline src/components/product/ProductCard/product-card.scss:26]:
```suggestion
    box-shadow: 0 0 0 1px var(--bg-neutral-lightest), 0 0 0 var(--border-width-3) var(--bg-focus-ring);
```

**Review by hellofanny — APPROVED** [review]
Niceee team 👏 👏 👏

File: src/components/product/ProductCard/product-card.scss (hunk 1 above)
Comment (hellofanny) [reply src/components/product/ProductCard/product-card.scss:26]: Actually. I just did it [here](https://github.com/vtex-sites/base.store/pull/219/commits/5ba320546e1a1dea64e8287b43bb1ae38b3dd724). Lemme know if you have any thoughts about it.

**Review by filipewl — APPROVED** [review]

**Review by eduardoformiga — APPROVED** [review]
✨ ✨ ✨

**Pull request merged**